dspace-api/src/main/java/org/dspace/workflow/TaskSet.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.workflow;
+
+import java.util.List;
+import javax.validation.constraints.NotNull;
+
+/**
+ * A collection of workflow tasks.
+ */
+public class TaskSet {
```

Comment:
Same here, improve javadocs please.  Again, the phrase "workflow tasks" is probably incorrect.  These are Workflow *Curation* Tasks.

Reply:
Right.  I've tried to make explicit which bits are Curation objects and which are Workflow objects.